infra/_modules/networking/main.tf

```diff
@@ -4,8 +4,8 @@
 
 module "snet" {
   source               = "github.com/pagopa/terraform-azurerm-v3//subnet?ref=v8.3.0"
-  name                 = "${var.project}-${var.application_basename}-srch-snet-001"
-  resource_group_name  = var.resource_group_name
+  name                 = "${var.project}-${var.application_basename}-srch-snet-01"
+  resource_group_name  = "${var.project}-common-rg-01"
```

Comment:
I'd restore the previous value where the resource group name is given by the caller, which has the actual reference

Reply:
It has been changed cause the creation of the snet would fail if the resource group differs from the one of the vnet:
<img width="1815" alt="image" src="https://github.com/pagopa/io-services-cms/assets/43080407/99e79a6b-43d0-4a55-99f6-6e045531d10b">